test/rules/typedef/arrow-call-signature&parameter/test.ts.lint

```diff
@@ -0,0 +1,12 @@
+class AComponent {
```

Comment:
I'm hesitant to introduce a "&" character into the folder path (call me paranoid) -- can you rename the folder to "arrow-call-signature-and-parameter"?